src/woocommerce/helpers.ts

```diff
@@ -0,0 +1,182 @@
+import { check, sleep } from 'k6'
+import http, { Response } from 'k6/http'
+import { parseHTML } from 'k6/html'
+
+import papaparse, { ParseResult } from 'papaparse'
+import slug from 'slugify'
+
+import {
+    map, flatMap, filter, get, uniq, parseInt,
+    replace, split, trim, toString, toLower,
+    has, isEmpty, isString
+} from 'lodash'
+
+
+const SHOP_URL = __ENV.SHOP_URL || 'http://localhost:8080/'
+
+//
+//  SAMPLE DATA
+
+export type Product = {
+    id: number
+    sku: string
+    name: string
+    categories: string[]
+}
+
+function isValidProduct(productData: object) {
+    return (
+        !isEmpty(get(productData, '\ufeffID'))
```

Comment:
`\ufeff` looks like a BOM https://github.com/clld/clldutils/issues/65. Maybe cleaning up the csv file will help getting rid of it.

Reply:
Yes, it could be removed, but I kinda thought it would be ok to have it as a drop in replacement from https://github.com/woocommerce/woocommerce/tree/master/sample-data  
Maybe we can make a PR over there to remove it.  
Or just remove it in our repo, like you suggested.